lib/msf/core/exploit/smb/client/psexec.rb

```diff
@@ -44,7 +44,7 @@ def service_name
   # Retrieve the SERVICE_DISPLAY_NAME option, generate a random
   # one if not already set.
   #
-  # @return service_display_name [String] the display name of the service.
+  # @return [String] service_display_name the display name of the service.
```

Comment:
`@return` doesn't recognize a name, so this will just end up as part of the description. Better would be to remove `service_display_name` entirely.